kratos/geometries/nurbs_curve_on_surface_geometry.h

```diff
@@ -287,10 +269,47 @@ class NurbsCurveOnSurfaceGeometry : public Geometry<typename TSurfaceContainerPo
             1e-6);
     }
 
+    /* @brief Provides the nurbs boundaries of the NURBS/B-Spline curve.
+     * @return domain interval.
+     */
+    NurbsInterval DomainInterval() const
+    {
+        return mpNurbsCurve->DomainInterval();
+    }
+
+    ///@}
+    ///@name Projection Point
```

Comment:
a small description would be helpful

Reply:
yo, comments are added. This is the block btw. I added it to the function itself.